packages/appeals-service-api/src/services/appeal.service.js

```diff
@@ -141,6 +150,74 @@ async function updateAppeal(id, appealUpdate) {
 	return updatedAppeal;
 }
 
+/**
+ * @param {*} appealUpdate - appeal to update
+ * @param {*} appeal - existing appeal
+ * @returns {Promise<void>}
+ */
+async function linkToUser(appealUpdate, appeal) {
+	const updateLinkForS76 =
```

Comment:
Meant to be S78?

Reply:
ugh I get this wrong all the time, used to work on 96 and now confuse the numbers